metaStoreApp/spec/factories.rb

```diff
@@ -0,0 +1,41 @@
+FactoryBot.define do
```

Comment:
better to have separate factory files for every models, not a big deal here since we have very small models

Reply:
okay